Add second batch of non-default thumbnails

SpartanFrederic104 opened the pull request:
This PR adds new thumbnails for the following vehicle types:
- Bumper Boats (Boat Hire)
- Cheshire Cats (Car Ride)
- Floorless Twister Trains (Twister Roller Coaster)
- Motorbikes (Steeplechase)
- Racing Cars (Car Ride)
- Roller Coaster Train (backwards) (Looping Roller Coaster)
- Steam Trains with Covered Cars (Miniature Railway)
- Streamlined Monorail Trains (Monorail)

This leaves the Canoes, Swans, Water Tricycles (all Boat Hire), and Spinning Cars (Classic Mini Roller Coaster) remaining without thumbnails. If you feel that any of my thumbnails should be changed, feel free to change them (or I can change them if you want).

SpartanFrederic104 committed:
Create images

SpartanFrederic104 committed:
Delete images

SpartanFrederic104 committed:
Update Bumper Boats thumbnail

Screenshot taken in Sweltering Towers custom scenario

SpartanFrederic104 committed:
Add Motorbikes thumbnail

Screenshot taken in Luminous Landscapes custom scenario

SpartanFrederic104 committed:
Add new Racing Cars thumbnail

Screenshot taken in Sunset Swamps custom scenario

SpartanFrederic104 committed:
Add Reversed Steel RC thumbnail from Wiki

SpartanFrederic104 committed:
Add Streamlined Monorail thumbnail

Screenshot taken in Heide-Park scenario

SpartanFrederic104 committed:
Add Floorless thumbnail

Screenshot taken from Sweltering Towers custom scenario

SpartanFrederic104 committed:
Add Covered Steam Trains thumbnail

Screenshot taken from Heide-Park scenario

SpartanFrederic104 committed:
Add Cheshire Cats thumbnail

Screenshot taken in Blackpool Pleasure Beach scenario (origin of RCT2 Cheshire Cats thumbnail)

spacek531 commented:
I'm sorry, but only one of these can be accepted as-is. The thumbnails ought to use only sprites available to the expansion they belong to (or older) and only two of them don't use RCT2 land textures among other anachronisms. The Cheshire cats ride is also the same as RCT2's, which while intentional, I would prefer a custom thumbnail, unless @ZeeMajora thinks it's appropriate.

SpartanFrederic104 commented:
Oh, now I see where I went wrong. I will redo the thumbnails entirely to remove the anachronisms and change the scenarios entirely. Would Crumbly Woods work for the Cheshire Cats?

SpartanFrederic104 committed:
Replace Cat Cars thumbnail

As spacek531 suggested, no anachronisms or copies of RCT2 thumbnails, so here's a new thumbnail in Crumbly Woods

SpartanFrederic104 committed:
Replace Racing Cars thumbnail

Use a new thumbnail based in Mel's World from the original RCT1

SpartanFrederic104 committed:
Replace Motorbikes thumbnail

New thumbnail taken in Fort Anachronism, an original RCT1 DLC scenario

SpartanFrederic104 committed:
Replace Streamlined Monorail thumbnail

Taken in RCT1 scenario White Water Park

SpartanFrederic104 committed:
Add new Steel RC reverse thumbnail

Screenshot taken in the RCT1 scenario Pacific Pyramids

SpartanFrederic104 committed:
Replace Bumper Boats thumbnail

Screenshot taken in White Water Park from the original RCT1

SpartanFrederic104 committed:
Replace Steam Trains thumbnail

Screenshot taken in RCT1 AA scenario Fruit Farm

SpartanFrederic104 committed:
Replace Floorless thumbnail

New screenshot taken in RCT1 AA scenario Fruit Farm

SpartanFrederic104 commented:
Okay, I've replaced all the thumbnails with new ones taken in scenarios that are in their respective game of origin (the ones labeled "rct1" are the original RCT1 scenarios, including the DLC Fort Anachronism, and "rct1.aa" are in the RCT1 Added Attractions/Corkscrew Follies scenarios).
![preview](https://user-images.githubusercontent.com/61885939/185725109-9c52acf4-e4eb-4573-a4b4-3853b485ea11.png) - taken in White Water Park
![preview](https://user-images.githubusercontent.com/61885939/185725112-216869ac-56b1-43a1-a89e-392353438b31.png) - taken in Crumbly Woods
![preview](https://user-images.githubusercontent.com/61885939/185725116-b1f94c20-1610-4a39-968c-d01131ce87b1.png) - taken in Fort Anachronism
![preview](https://user-images.githubusercontent.com/61885939/185725129-769f9952-2025-446c-83f4-e2adfd12698b.png) - taken in Mel's World
![preview](https://user-images.githubusercontent.com/61885939/185725133-a4d56bf3-3e7d-4f1e-8c42-a8279c39ed54.png) - taken in Pacific Pyramids
![preview](https://user-images.githubusercontent.com/61885939/185725139-0aeb7d46-43ba-4b45-8c19-e8317cdb0dc2.png) - taken in White Water Park
![preview](https://user-images.githubusercontent.com/61885939/185725082-d242521d-4372-4154-9f2d-bb5ed86f2325.png) - taken in Fruit Farm
![preview](https://user-images.githubusercontent.com/61885939/185725089-ffa2810e-baa3-49b1-98f4-383251af04c3.png) - taken in Fruit Farm
Do these look better to you now? For these non-default thumbnails, I had to avoid using Megaworld Park as much as possible since a good number of the RCT2 preview thumbnails originate from Megaworld Park and I wanted to use more of the RCT1 scenarios.

I feel like the new floorless thumbnail could be updated again, preferably to something like the new image below, since I don't like how the updated one turned out.
![preview](https://user-images.githubusercontent.com/61885939/185725220-1f8ecd7f-24ca-4fe3-a23f-808850f85ea7.png)

spacek531 commented:
You need to use opaque water and terrain gridlines without landscape smoothing. I prefer not taking pictures of existing rides in the game - I'd rather have original creations.

SpartanFrederic104 commented:
That I can do. If banners appear in the thumbnails, do you also want them to have their text displayed in uppercase like in RCT1?

spacek531 commented:
Probably best just to not have banners. You're making a tiny area that looks good - 112 x 112 pixels, it doesn't have to be too fancy.

SpartanFrederic104 committed:
New Bumper Boats thumbnail

With opaque water

SpartanFrederic104 committed:
Update Cheshire Cats thumbnail

With landscape gridlines and no landscape smoothing

SpartanFrederic104 committed:
New Motorbikes thumbnail

With landscape gridlines and no landscape smoothing

SpartanFrederic104 committed:
New Racing Cars thumbnail

With landscape gridlines, opaque water, and no landscape smoothing

SpartanFrederic104 committed:
New Steel RC Rev thumbnail

SpartanFrederic104 committed:
New Streamlined Monorail Thumbnail

SpartanFrederic104 committed:
New Floorless thumbnail

SpartanFrederic104 committed:
New Steam Trains thumbnail

SpartanFrederic104 commented:
I have updated the thumbnails once again, using the Mel's World and Trinity Islands for the RCT1 vehicles, and Adrenaline Heights scenarios for the two RCT1 AA/CF vehicles and also having the landscape gridlines and opaque water enabled and landscape smoothing disabled like you requested.
![preview](https://user-images.githubusercontent.com/61885939/185730165-a9bd4938-abb7-4fc3-ae13-3d21cbcbc63a.png)
![preview (1)](https://user-images.githubusercontent.com/61885939/185730167-40e13b52-a827-4302-827d-a83deb70c979.png)
![preview (2)](https://user-images.githubusercontent.com/61885939/185730168-fc548474-b57c-4ac9-8043-5fd5aacad043.png)
![preview (3)](https://user-images.githubusercontent.com/61885939/185730169-b523bbf9-24ea-4266-b23d-ffd8c9500ddc.png)
![preview (4)](https://user-images.githubusercontent.com/61885939/185730170-f2a354c8-2e07-45ca-b981-7381bb5e89f5.png)
![preview (5)](https://user-images.githubusercontent.com/61885939/185730171-913c680f-466c-4437-a0fe-584fd02261c2.png)
![preview (6)](https://user-images.githubusercontent.com/61885939/185730172-2b11dce0-642f-4206-aa50-954fe4e32bb2.png)
![preview (7)](https://user-images.githubusercontent.com/61885939/185730173-abc69219-53ee-43c6-a9bc-0d471a9e13bd.png)
And yes, all of these are original creations too in these scenarios.

spacek531 commented:
@ZeeMajora what do you think?

ZeeMaji commented:
I think the scenes look alright, the only one i'm a bit unsure of is the motor bikes, covered steam train & bumper boats thumbnails as i feel it is a bit barren or have odd car colors.
I would suggest adding some foliage to the covered steam trains thumbnail, how about some jungle trees with some dinosaur scenery as they were added in AA? As for the motorbikes i would suggest doing something different, i myself wanted to do something with martian rocks for that thumbnail so perhaps give that a try.
For the bumper boats i would just change their colors as i do feel the colors you chose for them are a bit samey, i would suggest two different colors per boat that contrast eachother nicely.

In terms of image table stuff the thumbnails also need to be added to the fallback (noCsgImages) image tables as it is our standard to use the RCT1 thumbnails for them.

I have also noticed some palette issues, for one the images with water in them use the "natural" water colors which will cause them to be animated ingame, and two the images with chain lifts in them use their color values which are not only non-functional ingame due to some jank with how pngs work but even if you used the proper values they'd animate just like the water.
For taking thumbnails i would suggest just using this custom palette i made which makes "natural" water use the static water colors, extract this json to your objects folder and use it and that should fix the water.
[extract_this_majora.water.wtrcyan.static.zip](https://github.com/spacek531/RCT1Vehicles/files/9425303/extract_this_majora.water.wtrcyan.static.zip)

To fix the lift hill colors you'll have to manually edit them in an external image editor as i don't think custom palettes can change the lift hill color, for them i use these two hex color values that came from the rct1 wooden twister thumbnail.
#233333
#5b3f1f
Seen here on the current reversed wooden trains thumbnail
![preview](https://user-images.githubusercontent.com/42477864/186685438-fc11b0ba-fd8e-428c-a49c-e27fab9c80b3.png)


I do have one request to revert the reversed looping RC thumbnail back to the pacific pyramids one as i feel it's a lot more unique.
![185725133-a4d56bf3-3e7d-4f1e-8c42-a8279c39ed54](https://user-images.githubusercontent.com/42477864/186680357-2ad4ffef-4916-4345-8783-1d09bae12f7c.png)
Either that or make a new egyptian themed scene for it as we don't have any egyptian themed thumbnails either from the original thumbnail batch or the new custom ones we're making for this project.

ZeeMaji commented:
And also, if you can compile openrct2 builds yourself i would highly recommend retaking these thumbnails in a special build i uploaded that i'm using for this project. It has a hack for RCT1 support rendering (rendering track supports for every tile) which will make them more accurate to how they'd actually look if you were to build them in RCT1 proper.
https://github.com/ZeeMajora/OpenRCT2/tree/rct1-project

spacek531 commented:
Just to let you know, I am pulling rank on the cat thumbnail. I built a ride for it a few months ago and I don't feel like rebuilding it.

The car ride I don't like being built directly on the water and using default colors for the cars and the junior coaster track.
The train I would like to see redone using Majora's suggestions.
There's some kind of compositional or color issue with the floorless train. I'd like that looked at.
I think the motorbike thumbnail is ok, although since we are running out of available RCT1 rides, it would be nice to include the martian texture.
The monorail thumbnail is good except for the colors, I think those need to be tweaked.
The bumper boats I would like if there was a little more land on the left side.

ZeeMaji commented:
I will also be taking over the reversed steel thumbnail.
